src/Controls.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import './controls.css';
+
+class Controls extends Component {
+  render() {
+    return (
+      <React.Fragment>
+      <div className="controls">
+        <div className="controls-title">Instructions to Play</div>
+        <ol>
+          <li>1. Press <i class="fas fa-caret-left fa-2x"></i> to Move Left.</li>
```

Comment:
Instead of importing using the whole Font Awesome library for 4 icons, we can just use reuse the SVG arrows added [here](https://github.com/itaditya/trick-or-treat-game/blob/master/src/Game.js#L283). For now you can just copy-paste the svg. Later on we can import the svg code at both places instead of duplicating.